modules/ossm-installing-istio-ambient-mode.adoc

```diff
@@ -13,6 +13,7 @@ You can install {istio} ambient mode on {ocp-product-title} 4.19 or later and {S
 * You have deployed a cluster on {ocp-product-title} 4.19 or later.
 * You have installed the {SMProduct} Operator 3.1.0 or later in the {ocp-product-title} cluster.
 * You are logged in to the {ocp-product-title} cluster either through the web console as a user with the `cluster-admin` role, or with the `oc login` command, depending on the installation method.
+* You have configured the OVN-Kubernetes CNI to use local gateway mode by setting the `routingViaHost` field as `true` in the `gatewayConfig` specification in the Cluster Network Operator. For more information, see "Configuring gateway mode"
```

Comment:
Is it possible to make the "Configuring gateway mode" a link, that points to the docs guide?